components/TechnologiesDetailsModal.tsx

```diff
@@ -0,0 +1,89 @@
+'use client';
+
+import { MoreHorizontal } from 'lucide-react';
+import Image from 'next/image';
+import React, { type FC, ReactElement } from 'react';
+
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/ui/dialog';
+import { Spinner } from '@/components/ui/spinner';
+import { Table, TableBody, TableCell, TableRow } from '@/components/ui/table';
+
+type Technology = {
+  name: string;
+  category: number;
+  icon: string;
+  index: string;
+  sourceUrl: string;
+  detectedTime: number;
+  latestDetectedTime: number;
+  website: string;
+  siteListUrl: string;
+  categoryString?: string;
+};
+
+type TechnologiesDetailsProps = {
+  technologies: Technology[];
+};
+
+const TechnologiesDetailsModal: FC<TechnologiesDetailsProps> = ({
+  technologies,
+}): ReactElement => {
+  const [open, setOpen] = React.useState(false);
+
+  return (
+    <>
+      <div
+        key={'more'}
+        onClick={() => setOpen(true)}
+        className={`flex h-12 w-12 justify-center rounded-lg hover:cursor-pointer hover:bg-black/5`}
+      >
+        <MoreHorizontal className="m-auto h-5 w-5 text-black" />
+      </div>
+
+      <Dialog modal open={open} onOpenChange={setOpen}>
+        <DialogContent>
+          <DialogHeader>
+            <DialogTitle>Used Technologies</DialogTitle>
+            <DialogDescription className={`max-h-96 overflow-scroll`}>
+              <Table>
+                <TableBody>
+                  {technologies.map((technology: Technology) => {
+                    return (
+                      <TableRow key={technology.index}>
+                        <TableCell>
+                          <Image
+                            src={`https://www.whatruns.com/imgs/${technology.icon}`}
+                            alt={technology.name}
+                            width={20}
+                            height={20}
+                          />
+                        </TableCell>
+                        <TableCell>
+                          <a
+                            className="cursor-pointer select-none decoration-slate-700 decoration-dotted underline-offset-4 hover:underline dark:decoration-slate-300"
+                            href={technology.website}
+                          >
+                            {technology.name}
+                          </a>
+                        </TableCell>
+                        <TableCell>{technology.categoryString}</TableCell>
+                      </TableRow>
+                    );
+                  })}
+                </TableBody>
```

Comment:
We should search for a solution without these scrollbars. At least horizontally.

![image](https://github.com/maaaathis/digga/assets/37186532/b19178da-0419-4552-9fd9-f2f553787fe5)